src/main/java/com/internet/shop/controller/cart/DeleteProductController.java

```diff
@@ -0,0 +1,34 @@
+package com.internet.shop.controller.cart;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.service.ProductService;
+import com.internet.shop.service.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/shopping-carts/products/delete")
+public class DeleteProductController extends HttpServlet {
```

Comment:
this is the next hw let's move it to the next pr

Reply:
I really had not red next hw before creating this class. Removed.